app/javascript/vue/shared/content/modal/move.vue

```diff
@@ -77,6 +77,16 @@
       },
       cancel() {
         $(this.$refs.modal).modal('hide');
+      },
+      getOptions(targets) {
+        const updatedTargets = targets.map(target => {
+          if (target[1] === null) {
+            target[2] = this.i18n.t('protocols.steps.modals.move_element.result.untitled_result');
+          }
+          return target;
+        });
+
+        return updatedTargets;
```

Comment:
Move this to a computed property called `targetOptions`. You can also just directly return the result of the map `return targets.map(...`.